modules/gltf/skin_tool.cpp

```diff
@@ -563,9 +563,12 @@ Error SkinTool::_create_skeletons(
 		Vector<Ref<GLTFNode>> &nodes,
 		HashMap<ObjectID, GLTFSkeletonIndex> &skeleton3d_to_gltf_skeleton,
 		Vector<Ref<GLTFSkeleton>> &skeletons,
-		HashMap<GLTFNodeIndex, Node *> &scene_nodes) {
+		HashMap<GLTFNodeIndex, Node *> &scene_nodes,
+		int p_naming_version) {
+	HashSet<String> unique_node_names = unique_names.duplicate();
```

Comment:
You can use the copy constructor:

```suggestion
	HashSet<String> unique_node_names(unique_names);
```

This should allow you to avoid modifying `hash_set.h` :)

Reply:
It's worth a thought as a general codebase change, but out of scope for the PR.